packages/desktop/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "quiet",
+  "name": "@quiet/desktop",
```

Comment:
I like this since we will now have @quiet/desktop and @quiet/mobile. I think it's more clear

Reply:
Yes! I only want to make sure it won't break anything so whoever wants to jump in and help me with testing, please do